src/client/core/request-barrier.js

```diff
@@ -30,46 +30,61 @@ export default class XhrBarrier extends EventEmitter {
     }
 
     _init () {
-        var onXhrSend      = e => this._onXhrSend(e.xhr);
-        var onXhrCompleted = e => this._onXhrCompleted(e.xhr);
-        var onXhrError     = e => this._onXhrError(e.xhr, e.err);
+        var onXhrSend      = e => this._onRequestSend(e.xhr);
```

Comment:
`onXhrSend`, `onXhrCompleted`, `onXhrError` are used only for xhr events.
`_onRequestSend`, `_onRequestCompleted`, `_onRequestError` are common internal functions.  Used for xhr and fetch.
So I think we should not rename vars.